test/glific/contacts_field_test.exs

```diff
@@ -101,6 +119,40 @@ defmodule Glific.ContactsFieldTest do
              Contacts.get_contact(contact.id)
   end
 
+  test "update_contacts_field/2 with valid data updates the contacts_field when scope is group",
+       %{
+         organization_id: organization_id
+       } do
+    wa_managed_phone = Fixtures.wa_managed_phone_fixture(%{organization_id: organization_id})
+
+    wa_group =
+      Fixtures.wa_group_fixture(%{
+        organization_id: organization_id,
+        wa_managed_phone_id: wa_managed_phone.id
+      })
+      |> ContactField.do_add_wa_group_field("test", "test field", "hello")
+
+    assert wa_group.fields != %{}
+
+    attrs =
+      %{
+        name: "Citizen",
+        shortcode: "citizen",
+        organization_id: organization_id
+      }
+
+    contacts_field = ContactsField |> where([cf], cf.shortcode == "test") |> Repo.one()
+
+    assert {:ok, %ContactsField{} = updated_contacts_field} =
+             ContactField.update_contacts_field(contacts_field, attrs)
```

Comment:
since its used outside contact also might need to rethink if we should still call it contact field(I know we have setup scope and there is few things we don't want to copy the same code)